app/code/Magento/Checkout/view/frontend/web/js/view/billing-address.js

```diff
@@ -52,6 +52,10 @@ function (
         },
         countryData = customerData.get('directory-data'),
         addressOptions = addressList().filter(function (address) {
+            if (address.isDefaultBilling()) {
```

Comment:
Hi @ravi-chandra3197. Thanks for collaboration.  You set default billing address in the filter callback function. That's mean you set address multiple times because the `filter` method iterate array like foreach. I think the better way is move sets address functionality outside of the filter method.